enarx-keep/shims/shim-sev/.cargo/config

```diff
@@ -2,5 +2,6 @@
 target = "x86_64-unknown-linux-musl"
 rustflags = [
     "-C", "relocation-model=pic",
+    "-C", "link-args=-Wl,--sort-section=alignment",
```

Comment:
This should be a separate PR. We can merge this immediately.

Reply:
will be, after this one